File: nodejs/seguranca-ms/src/core/user/dto/change-password-request.ts
export class ChangePasswordRequest {

    constructor (userId: string,
            password: string,
            newpassword: string) {
        this.userId = userId;
        this.password = password;
        this.newpassword = newpassword;
    }

    userId: string;
    password: string;
    newpassword: string;

}

File: java/quarkus/seguranca-ms/src/main/java/br/ejcb/cfp/seguranca/api/dto/ChangePasswordRequest.java
package br.ejcb.cfp.seguranca.api.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.With;

@Getter
@Setter
@With
@NoArgsConstructor
@AllArgsConstructor
public class ChangePasswordRequest {

	private String password;
	private String newPassword;
	
}
